markdown/2017-10-02-welcome.md

```diff
@@ -0,0 +1,380 @@
+
+Come and visit us at our stall at the Freshers' Fair to learn more about us,
+chat with the committee, eat lots of sweets, and win some great prizes in our
+speed typing competition! We'll be there all day so make sure to check out all
+of the other fantastic opportunities as well! You can find us by consulting the
+official map.
+
+The speed typing prizes are (in no particular order):
+- GoPro HERO5 Session
+- 25" 21:9 LG UltraWide Monitor
+- Amazon Echo
+
+# Buddies Lunch
+
+- **When:** 12.30-14.00, 9th October 2017
+- **Where:** Huxley 341/342
+
+As a fresher, you will have already heard of our Mums and Dads (MaD) programme
+from your department representatives. If you don't have any parents assigned yet
+or have never heard of this programme, please reach out to your department
+representatives (above).
+
+The buddies lunch will be an opportunity for you to meet your MaD parents and
+have lunch with them! We will be offering free food and drink and issuing a
+number of challenges. You might even win a prize! And if you're a parent, you'll
+get to meet your children!
+
+# Sponsors' Exhibition
+
+- **When:** 14.00-17.30, 12th October 2017
+- **Where:** The Great Hall
+- **Link:** [Facebook](https://www.facebook.com/events/1852626871714356/)
+
+Looking for an internship? Or want to talk directly to industry and our fabulous
+sponsors? Come join us in the Great Hall to have a chat with companies ranging
+from Facebook to Palantir and Netcraft to Microsoft in our premier autumn event.
+Bring lots of questions and be prepared to go home with a lot of swag!
+
+# Bloomberg CodeCon
+
+- **When:** 18.00-20.00, 13th October 2017
+- **Where:** Skempton 060
+- **Link:** [Facebook](https://www.facebook.com/events/1816220712002800/)
+
+CodeCon is an annual coding competition put on by Bloomberg and run all around
+the world, including at Imperial. Come for fun or competition, and if you do
+well enough you cold make it to the global finals! Free food and drink will be
+provided.
+
+---
+
+Some notes from our sponsors:
+
+# Opportunities at Netcraft
+
+<center>
+  <div style="position:relative;width:50%">
+    ![Netcraft Logo](https://docsoc.co.uk/img/sponsors/netcraft_logo.png)
+  </div>
+</center>
+
+Netcraft is a world-leading security services company, performing penetration
+testing and anti-fraud services to critical institutions throughout the world.
+
+We're currently looking to recruit software developers to our offices in Bath,
+England. We are keen to receive applications for graduate positions, industrial
+placements, or summer internships from high performing Computing students.
+
+Projects involve working on a range of commercial services extending Netcraft's
+network exploration and internet security services.
+
+To find out more, take a look at our recruitment booklet:
+<https://www.netcraft.com/about-netcraft/jobs/NetcraftGraduateBooklet.pdf>.
+
+Interested? Send a CV to <cv@netcraft.com>.
+
+# Bloomberg Blog Post
```

Comment:
Are you only putting the logo for Netcraft? Why not the other companies too.

Reply:
Netcraft and SIG asked for them